mobilecoind/src/service.rs

```diff
@@ -2173,6 +2276,7 @@ build_api! {
 
     // Signed contingent inputs
     generate_swap GenerateSwapRequest GenerateSwapResponse generate_swap_impl,
+    generate_mixed_tx GenerateMixedTxRequest GenerateMixedTxResponse generate_mixed_tx_impl,
```

Comment:
i still need to write some tests for this API

Reply:
okay, have a test, but should make more tests still

Reply:
I think I should also try to make it so that specifying the utxo list is not required to use this function, and the server will figure that out for you if you don't specify